package.json

```diff
@@ -26,7 +26,7 @@
   "license": "MIT",
   "dependencies": {
     "@actions/core": "1.2.6",
-    "typed-rest-client": "1.7.3",
+    "typed-rest-client": "1.8.4",
```

Comment:
This update is necessary because [version 1.7.3](https://github.com/microsoft/typed-rest-client/tree/e4b9bf2a73677c56e124ecc963ef7a8677d3d738/) has no retry feature for network errors.
It was introduced by https://github.com/microsoft/typed-rest-client/commit/06d0fc1eaead5e20ae687b5c844996a37c24e7b2 and released as a part of version 1.8.0.